### ISSUE TITLE
Generalize the /plugin page

### ISSUE THREAD

**Issue by cchristelis** [kartoza/plugin-host#2]
# Problem

The /plugin page is static, regardless of which projects have been loaded. 
# Proposed Solution

Create this page dynamically based on the entries in the Plugin and Release tables.

**Comment by dimasciput** [comment]
Yes, and I think it should be generated from the latest version of Plugin. Are all plugins going to be qgis plugin? @cchristelis

**Comment by cchristelis** [comment]
So I think the goal of this is to be a QGIS plugin manager. I suppose we could try and generalize it further. Would we define the way the /plugin (or in this case the /qgis-plugin) url renders based on another model configuration?
